### PR TITLE
chore: improve debuggability of identify protocol

### PR DESCRIPTION
**Pull request by keroro520**
### What is changed and how it works?

What's Changed: Update some logs related to the "identify" network protocol.

### Check List 

Tests 

- No code ci-runs-only: [ quick_checks,linters ]

### Release note 

```release-note
None: Exclude this PR from the release note.
```

### CODE REVIEW

**Review by driftluo — APPROVED** [review]

File: network/src/protocols/identify/mod.rs [hunk 1]
```diff
@@ -541,7 +538,11 @@ impl Callback for IdentifyCallback {
         MisbehaveResult::Continue
     }
 
-    fn misbehave(&mut self, _peer_id: &PeerId, _kind: Misbehavior) -> MisbehaveResult {
+    fn misbehave(&mut self, session: &SessionContext, reason: Misbehavior) -> MisbehaveResult {
+        error!(
+            "IdentifyProtocol detect misbehave, session: {:?}, reason: {:?}",
```
Comment (quake) [inline network/src/protocols/identify/mod.rs:543]:
```suggestion
            "IdentifyProtocol detects abnormal behavior, session: {:?}, reason: {:?}",
```

File: network/src/protocols/identify/mod.rs [hunk 2]
```diff
@@ -582,7 +583,10 @@ impl Identify {
 
         let name = reader.name().as_utf8().ok()?.to_owned();
         if self.name != name {
-            debug!("Not the same chain, self: {}, remote: {}", self.name, name);
+            warn!(
+                "IdentifyProtocol detect peer has different network identifiers, local network id: {}, remove network id: {}",
```
Comment (quake) [inline network/src/protocols/identify/mod.rs:587]:
```suggestion
                "IdentifyProtocol detects peer has different network identifiers, local network id: {}, remote network id: {}",
```

File: network/src/protocols/identify/mod.rs (hunk 1 above)
Comment (keroro520) [reply network/src/protocols/identify/mod.rs]: I corrected the sentences as you suggested in the last force-push, thanks.

**Review by quake — APPROVED** [review]

**Comment by keroro520** [comment]
bors r=quake,driftluo

**Pull request merged**